docs/conf.py

```diff
@@ -103,15 +103,13 @@
     r'http://localhost:\d+/',  # local URLs
     r'https://codecov\.io/gh/cherrypy/cheroot/branch/master/graph/badge\.svg',
     r'https://github\.com/cherrypy/cheroot/actions',  # 404 if no auth
-
```

Comment:
Removing separators between sections makes the grouping unobvious. If it's really desired, those bits of data should be put into named variables and merged here, to preserve relaying the original intent.